teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] Similar to the implementation for the student, ensure you add the remaining parameters to the `initialize` constructor

Reply:
Okay! :+1:  :rocket: